config/default/config_default.yaml

```diff
@@ -1,4 +1,5 @@
 agent_id: "IAI MovieBot"
+simulator_id: "User Simulator"
```

Comment:
Perhaps add the version here as well?

Reply:
I would argue against it as the version may change but the default configuration should work with "any" version.